docs/ref/databases.txt

```diff
@@ -169,6 +169,19 @@ lookups that use the ``LIKE`` operator in their SQL, as is done with the
 
 .. _PostgreSQL operator class: http://www.postgresql.org/docs/current/static/indexes-opclass.html
 
+Speed up test execution times with non-durable settings
+-------------------------------------------------------
+
+It is possible to speed up test execution times by configuring PostgreSQL to be non durable.
+
+.. warning::
+
+    This is dangerous - it could make your database more likely to lose data or become corrupted, only use on a development machine where you can easily restore the entire contents of all databases in the cluster.
```

Comment:
What about:

"This is dangerous: it will make your database more likely to lose data or become corrupted in case of server crash or power loss, only use on a development machine where you can easily restore the entire contents of all databases in the cluster."